src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/ccd/CcdClient.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests.ccd;
+
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+@Service
+public class CcdClient {
+
+    private static final Logger log = LoggerFactory.getLogger(CcdClient.class);
+
+    private static final String AWAITING_DCN_PROCESSING_FIELD_NAME = "awaitingPaymentDCNProcessing";
+    private static final String COMPLETE_AWAITING_DCN_PROCESSING_EVENT_ID = "completeAwaitingPaymentDCNProcessing";
+
+    private final CoreCaseDataApi ccdApi;
+    private final CcdAuthenticatorFactory authenticatorFactory;
+
+    public CcdClient(
+        CoreCaseDataApi ccdApi,
+        CcdAuthenticatorFactory authenticatorFactory
+    ) {
+        this.ccdApi = ccdApi;
+        this.authenticatorFactory = authenticatorFactory;
+    }
+
+    public void completeAwaitingDcnProcessing(
```

Comment:
why do you want to call it from tests?

Reply:
It is really not necessary, but this PR is mainly about harness to send requests to CCD.